kairon/shared/channels/whatsapp/bsp/gupshup.py

```
@@ -503,6 +560,12 @@ def get_broadcast_template_params(self, raw_template, template_config):
             container_meta = json.loads(raw_template.get("containerMeta", "{}"))
         except Exception:
             container_meta = {}
+        try:
+            parsed_data = json.loads(template_config.get("data", "[]") or "[]")
+        except Exception:
```

Ruff (0.16.1)

[warning] Do not catch blind exception: `Exception`

(BLE001)